lib/hooks/api/useContributorPullRequests.ts

```diff
@@ -32,6 +33,10 @@ export function getContributorPRUrl(
     query.set("repoIds", repoIds.join(","));
   }
 
+  if (mostRecent) {
+    query.set("rangeType", "all");
+  }
+
   query.set("range", `${range}`);
```

Comment:
Is `range` still required if we're saying get all via

```javascript
  if (mostRecent) {
    query.set("rangeType", "all");
  }
```
?

Reply:
Good catch, fixed